pom.xml

```diff
@@ -771,6 +771,19 @@
                 <artifactId>legend-engine-xt-relationalStore-spanner-grammar</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.finos.legend.engine</groupId>
+                <artifactId>legend-engine-xt-relationalStore-spanner-jdbc-shaded</artifactId>
+                <version>${project.version}</version>
+                <!-- need to exclude everything as long is this submodule is not extracted to separate project
```

Comment:
Use this as comment: we need exclusion to satisfy the enforcer plugin, when shaded dependency is a module within the project, as enforcer (incorrectly) relies on module's source pom.xml instead of it's published pom.xml. As expected, the problem goes away when the same dependency comes from a different project.